test/builders/build/eth-api/libraries/viem/send-transaction.js

```diff
@@ -0,0 +1,77 @@
+import { assert } from 'chai';
+import { createPublicClient, createWalletClient, http, formatEther, parseEther } from 'viem';
+import { privateKeyToAccount } from 'viem/accounts';
+import { moonbeamDev } from 'viem/chains';
+
+const fromAddressPrivateKey = "0x5fb92d6e98884f76de468fa3f6278f8807c48bebc13595d45af5bdc4da702133"
+const fromAddress = "0xf24FF3a9CF04c71Dbc94D0b566f7A27B94566cac"
+const toAddress = "0x73328873A6Ac3DFc9455986358F5230cB3B8e92c"
```

Comment:
the `toAddress` should be randomly generated, i'm not sure exactly how to do that with viem off-hand but I can look into it if you need any help! 🙂